assets/src/apps/authoring/components/AdaptivityEditor/ConditionItemEditor.tsx

```diff
@@ -120,7 +122,11 @@ const ConditionItemEditor: React.FC<ConditionItemEditorProps> = (props) => {
           placeholder="Target"
           value={fact}
           onChange={(e) => setFact(e.target.value)}
-          onBlur={(e) => handleFactChange(e.target.value)}
+          onFocus={(e) => dispatch(setCurrentPartPropertyFocus({ focus: false }))}
```

Comment:
If user is trying to modify / delete a condition editor name, we should not trigger the part component delete prompt.